### PR TITLE
Read homebrew-brewfile/Brewfile from API

### PR DESCRIPTION
**Pull request by isamuelson**
Use the GitHub API to read the homebrew-brewfile/Brewfile to to allow access to private repositories.

### CODE REVIEW

File: bin/strap.sh [hunk 1]
```diff
@@ -274,7 +274,9 @@ fi
 
 if [ -n "$STRAP_GITHUB_USER" ]; then
   REPO_URL="https://github.com/$STRAP_GITHUB_USER/homebrew-brewfile"
-  STATUS_CODE=$(curl --silent --write-out "%{http_code}" --output /dev/null $REPO_URL/blob/HEAD/Brewfile)
+  REPO_API_URL="https://api.github.com/repos/$STRAP_GITHUB_USER/homebrew-brewfile"
+  STATUS_CODE=$(curl -u $STRAP_GITHUB_USER:$STRAP_GITHUB_TOKEN --silent --write-out "%{http_code}" --output /dev/null $REPO_API_URL/contents/Brewfile)
```
Comment (mikemcquaid) [inline bin/strap.sh]: Wrap `$STRAP_GITHUB_USER:$STRAP_GITHUB_TOKEN` in double-quotes.

**Comment by mikemcquaid** [comment]
@isamuelson Could you make a more readable commit message and subject? I'd suggest.

```
Read homebrew-brewfile/Brewfile from API

Use the GitHub API to read the homebrew-brewfile/Brewfile to to allow access to private repositories.
```

**Comment by isamuelson** [comment]
This failure is not related to the PR

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Strap!

**Pull request merged**